src/main/java/core/basesyntax/service/FruitList.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.FruitTransaction;
+import java.util.List;
+
+public interface FruitList {
```

Comment:
The name 'FruitList' doesn't accurately represent the purpose of the interface. Consider renaming it to better reflect its functionality, such as 'FruitTransactionProcessor' or something similar that indicates it's processing a list of fruit transactions rather than just representing a list of fruits.